### PR TITLE
Add source info into callback

### PR DESCRIPTION
**Pull request by mfojtik**
Fixes: https://github.com/openshift/source-to-image/issues/66

### CODE REVIEW

**Comment by mfojtik** [comment]
@csrwng @soltysh PTAL

**Comment by soltysh** [comment]
LGTM

File: pkg/util/callback.go [hunk 1]
```diff
@@ -39,6 +39,10 @@ func (c *callbackInvoker) ExecuteCallback(callbackURL string, success bool, mess
 		"success": success,
 	}
 
+	for name, value := range labels {
+		d[name] = value
+	}
```
Comment (csrwng) [inline pkg/util/callback.go]: Is it necessary to keep the json that we send flat? Why not add a labels map under the main map?

Comment (mfojtik) [inline pkg/util/callback.go]: @csrwng flat json is more easy to parse ;-) but yeah, I can add this into a labels field

Comment (mfojtik) [inline pkg/util/callback.go]: @csrwng fixed

Comment (csrwng) [inline pkg/util/callback.go]: My concern was that at some point the labels would conflict with the other entries we add to the map

**Comment by csrwng** [comment]
LGTM

File: pkg/util/callback.go (hunk 1 above)
Comment (mfojtik) [inline pkg/util/callback.go]: @csrwng right, especially if we allow users to set arbitrary label

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**